packages/web-react/src/components/Toast/README.md

```diff
@@ -263,13 +264,108 @@ const [isOpen, setIsOpen] = React.useState(false)
+```
+
+### Minimal props
+
+```jsx
+const { showToast } = useUncontrolledToast(); // must be inside ToastProvider
+
+<ToastProvider>
+  <Button type="button" onClick={() => showToast(ToastTextWithLink, 'toast-id')}>
+    Show Toast
+  </Button>
+
+  <UncontrolledToast />
+</ToastProvider>;
+```
+
+### All possible props
+
+```jsx
+const { showToast } = useUncontrolledToast(); // must be inside ToastProvider
+
+<ToastProvider>
+  <Button
+    type="button"
+    onClick={() => showToast('Toast message', 'toast-id', {
+      autoClose: true
+      color: 'danger'
+      iconName: 'download'
+    })}
+  >
+    Show Toast
+  </Button>
+
+  <UncontrolledToast alignmentX="right" alignmentY="top" closeLabel="Close toast" hasIcon isDismissible />
+</ToastProvider>
+```
+
+### useUncontrolledToast hook
+
+Hook that provides information on `UncontrolledToast` and functions for opening and closing.
+What is uncontrolled component you can find [here][react-uncontrolled]
+
+This hook returns:
+
+| Name        | Type                                                         | Default    | Description                                              |
+| ----------- | ------------------------------------------------------------ | ---------- | -------------------------------------------------------- |
+| `autoClose` | `bool`                                                       | `false`    | Turn on/off auto close of UncontrolledToast after 5000ms |
+| `close`     | `function`                                                   | () => {}   | Function that will close UncontrolledToast               |
+| `color`     | [[Emotion Color dictionary][dictionary-color] \| `inverted`] | `inverted` | Color variant                                            |
+| `iconName`  | `string`                                                     | —          | Name of a custom icon to be shown along the message      |
+| `id`        | `string`                                                     | `''`       | ToastBar ID                                              |
+| `isOpen`    | `bool`                                                       | `false`    | Open state of UncontrolledToast                          |
+| `message`   | [`string` \| `ReactNode`]                                    | null       | Message inside UncontrolledToast                         |
+| `showToast` | `function`                                                   | () => {}   | Function that will open UncontrolledToast                |
+
+#### How to use `showToast` function:
+
+This function has two required parameters: message and ID.
+All other options are not required and can be omitted entirely.
+
+```jsx
+              ┌─⫸ Message inside UncontrolledToast (required)
+              │
+              │                 ┌─⫸ ToastBar ID (required)
+              │                 │
+showToast('Toast message', 'toast-id', {
+  autoClose: true         // Automatically close toast after 5000ms, default: false
```

Comment:
you are probably missing commas in this object.

Reply:
commas added